openapi/openapi.yaml

```diff
@@ -34,16 +34,16 @@ info:
       "Accept": "application/json",
       "Content-Type": "application/json"
     }
-    req = requests.request('PATCH', 'https://demo.schoolbox.com.au/api/user/1', json=data, headers=headers)
+    req = requests.request('PATCH', 'https://demo.schoolbox.education/api/user/1', json=data, headers=headers)
```

Comment:
I can't get https://demo.schoolbox.education to load or respond to an API request, does this actually work?

Reply:
curl: (35) error:0A000438:SSL routines::tlsv1 alert internal error

and

![image](https://github.com/user-attachments/assets/3d4c8151-6573-4c71-a41b-6634c5dca9e1)